MekHQ/data/mechfiles/mechs/3039u/Clint CLNT-2-4T.mtf

```diff
@@ -153,7 +153,7 @@ Overview: Not every BattleMech developed for the Star League was a  technologica
 
 Capabilities: The Clint mostly operates as a heavier recon 'Mech or lower-end skirmisher. In these roles it serves well, and it demands respect from common scouting designs such as the Stinger or Wasp. The Sloane 220 targeting computer means that its firepower hits more often than average, giving the Clint a higher punch than what might be thought at first sight.
 
-Deployment: The Clint was found in every army in the Inner Sphere during the Succession Wars. No factories produced the design, but notable concentrations were found in the Federated Commonwealth and Capellan Confederation. Certain other states, such as the Free Worlds League, often used their Clints for little more than spare parts. A modification made by the two major users of the design, the -2-4T sacrificed manueverability and short-ranged firepower in order to increase the number of autocannons it carried. Commonly used against VTOLs, tanks, and other conventional forces, it posed little threat against 'Mechs able to catch up to it. Just like the base Clint, no -2-4T's remained in the armies of House Davion of Liao (or any other Houses) by the end of the Jihad.
+Deployment: The Clint was found in every army in the Inner Sphere during the Succession Wars. No factories produced the design, but notable concentrations were found in the Federated Commonwealth and Capellan Confederation. Certain other states, such as the Free Worlds League, often used their Clints for little more than spare parts. A modification made by the two major users of the design, the -2-4T sacrificed manoeuvrability and short-ranged firepower in order to increase the number of autocannons it carried. Commonly used against VTOLs, tanks, and other conventional forces, it posed little threat against 'Mechs able to catch up to it. Just like the base Clint, no -2-4T's remained in the armies of House Davion of Liao (or any other Houses) by the end of the Jihad.
```

Comment:
Doing a quick look shows this changes the definition of sentence and is not an actual spelling error.

Reply:
Disregard this one. Both are valid.

Reply:
`manueverability` is a typo, `manoeuvrability` is the correct spelling.

Reply:
It is a typo but one of a valid variation of the word: https://www.merriam-webster.com/dictionary/maneuverability